src/components/controllers/doc-page.jsx

```diff
@@ -20,29 +20,27 @@ export function DocPage() {
 		return <NotFound />;
 	}
 
-	return <DocLayout />;
+	return <DocLayout isGuide />;
 }
 
-export function DocLayout() {
+export function DocLayout({ isGuide = false }) {
 	const { path } = useLocation();
 	const [lang] = useLanguage();
 
 	const { html, meta } = useContent([lang, path === '/' ? 'index' : path]);
 	useTitle(meta.title);
 	useDescription(meta.description);
 
-	const hasSidebar = meta.toc !== false && isDocPage(path);
```

Comment:
Bit redundant; `toc: false` is only set for the home page (despite `/blog`, `/about/*`, `/branding`, etc. also all making use of this component) and `isDocPage` was a regex for `^/guide`.

The `isGuide` prop is a simpler replacement, we can use the router to instead pass that information in rather than using a regex pattern.